scanners/zap-advanced/scanner/zapclient/spider/zap_spider_ajax.py

```diff
@@ -194,6 +194,16 @@ def configure_spider(self, spider_config: collections.OrderedDict):
                 method_name="set_option_random_inputs",
             )
 
+        if self._is_not_empty_integer("failIfFoundUrlsLessThan", spider_config):
+            self.failIfFoundUrlsLessThan = spider_config["failIfFoundUrlsLessThan"]
+        else:
+            self.failIfFoundUrlsLessThan = 0  # Default value
+        
+        if self._is_not_empty_integer("failIfFoundUrlsLessThan", spider_config):   
```

Comment:
This must be the configuration key `warnIfFoundUrlsLessThan`

Reply:
Fixed :)